templates/partial/_careers-job-list.html

```diff
@@ -20,12 +20,15 @@
     <input type="checkbox" name="roles" id="{{ job.id }}" value="{{ job.id }}" />
     <label class="p-checkbox-label--h2 u-no-padding--top" for="{{ job.id }}">
     {% endif %}
-      <h2 class="p-heading--three {% if checkbox %}u-sv3{% else %}u-no-padding--top u-sv1{% endif %}">
-        <a class="p-link--soft" href="/careers/{{ job.id }}">{{ job.title }}&nbsp;&rsaquo;</a>
+      <h2 class="p-heading--three {% if checkbox %}u-sv3{% else %}u-no-padding--top{% endif %}{% if job.description %} u-no-margin--bottom{% else %} u-sv1{% endif %}">
+        <a class="p-link--soft" href="/careers/{{ job.id }}/{{ job.joburl }}">{{ job.title }}&nbsp;&rsaquo;</a>
```

Comment:
This is good but could we add the word "suffix" in there? It's not the entire URL. Also, it's always attached to a "job" object right? Probably don't need to repeat the word "job", is "url_suffix" ok?

Reply:
fixed